Error compiling assets

-   Preflight Version: 1.7.2
-   PHP Version: 7.4.13 / 8.0.1
-   Database Driver & Version: Any

### Description:

**Module not found:** Error: Can't resolve `'@/Views/Components/Sections/ActionSectionBorder'` in `'/Users/thavarshan/Codebase/web/ignite/resources/js/Views/API'`.

```bash
ERROR in ./resources/js/Views/API/ApiTokenManager.vue?vue&type=script&lang=js& (./node_modules/babel-loader/lib/index.js??clonedRuleSet-5[0].rules[0].use[0]!./node_modules/vue-loader/lib/index.js??vue-loader-options!./resources/js/Views/API/ApiTokenManager.vue?vue&type=script&lang=js&) 169:0-76
Module not found: Error: Can't resolve '@/Views/Components/Sections/ActionSectionBorder' in '/Users/thavarshan/Codebase/web/ignite/resources/js/Views/API'

ERROR in ./resources/js/Views/Layouts/AppLayout.vue?vue&type=script&lang=js& (./node_modules/babel-loader/lib/index.js??clonedRuleSet-5[0].rules[0].use[0]!./node_modules/vue-loader/lib/index.js??vue-loader-options!./resources/js/Views/Layouts/AppLayout.vue?vue&type=script&lang=js&) 62:0-76
Module not found: Error: Can't resolve '@/Views/Components/Sections/ActionSectionBorder' in '/Users/thavarshan/Codebase/web/ignite/resources/js/Views/Layouts'

webpack compiled with 2 errors
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ development: `mix`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ development script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/thavarshan/.npm/_logs/2021-02-07T13_42_56_999Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ dev: `npm run development`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/thavarshan/.npm/_logs/2021-02-07T13_42_57_057Z-debug.log
```
### Steps To Reproduce:

run `npm i && npm run dev` after installing preflight in a new Laravel application.

--- a/stubs/resources/js/Views/API/ApiTokenManager.vue
+++ b/stubs/resources/js/Views/API/ApiTokenManager.vue
@@ -162,13 +162,13 @@
 </template>
 
 <script>
-import ActionSection from '@/Views/Components/Sections/ActionSection';
 import AppInput from '@/Views/Components/Inputs/Input';
-import Checkbox from '@/Views/Components/Inputs/Checkbox';
 import AppButton from '@/Views/Components/Buttons/Button';
-import ActionMessage from '@/Views/Components/Alerts/ActionMessage';
-import SectionBorder from '@/Views/Components/Sections/ActionSectionBorder';
+import Checkbox from '@/Views/Components/Inputs/Checkbox';
 import DialogModal from '@/Views/Components/Modals/DialogModal';
+import ActionMessage from '@/Views/Components/Alerts/ActionMessage';
+import ActionSection from '@/Views/Components/Sections/ActionSection';
+import SectionBorder from '@/Views/Components/Sections/SectionBorder';
 
 export default {
     components: {

--- a/stubs/resources/js/Views/Layouts/AppLayout.vue
+++ b/stubs/resources/js/Views/Layouts/AppLayout.vue
@@ -60,7 +60,6 @@ import Navbar from '@/Views/Components/Navbars/Navbar';
 import NavbarLink from '@/Views/Components/Navbars/NavbarLink';
 import Dropdown from '@/Views/Components/Dropdowns/Dropdown';
 import DropdownLink from '@/Views/Components/Dropdowns/DropdownLink';
-import SectionBorder from '@/Views/Components/Sections/ActionSectionBorder';
 
 export default {
     components: {

